File: core/src/test/java/cn/enaium/mineconf/java/MineConfServiceImpl.java
package cn.enaium.mineconf.java;

import cn.enaium.mineconf.MineConf;
import cn.enaium.mineconf.MineConfService;

/**
 * @author Enaium
 */
public class MineConfServiceImpl implements MineConfService {

    public final static MineConfTest.Config CONFIG = new MineConfTest.Config();
    public final static MineConf MINE_CONF = new MineConf("test", "Test").register(CONFIG);

    @Override
    public MineConf conf() {
        return MINE_CONF;
    }
}

File: core/src/test/kotlin/cn/enaium/mineconf/kt/MineConfServiceImpl.kt
package cn.enaium.mineconf.kt

import cn.enaium.mineconf.MineConf
import cn.enaium.mineconf.MineConfService
import cn.enaium.mineconf.java.MineConfTest

/**
 * @author Enaium
 */
class MineConfServiceImpl : MineConfService {
    override fun conf(): MineConf {
        return MINE_CONF
    }

    companion object {
        val CONFIG: MineConfTest.Config = MineConfTest.Config()
        val MINE_CONF: MineConf = MineConf("test", "Test").register(CONFIG)
    }
}
